src/replication_manager.py

```diff
@@ -90,10 +93,10 @@ def update_replication_type(self, db_state, ha_replics):
         else:
             logging.info("Here we should turn synchronous replication on.")
             if self._db.change_replication_to_sync_host(holder_fqdn):
-                logging.info('Turned synchronous replication ON.')
+                logging.info('ACTION. Turned synchronous replication ON.')
```

Comment:
`ACTION` should be in https://github.com/yandex/pgconsul/pull/59/files#diff-85b283c50c288107310ead558a64a812605cafbb275800211c01540f2c8fd98fR94 log message, before performing action, not after.